### ISSUE TITLE
Add Plan 9 licenses

### ISSUE THREAD

**Issue by tv42** [google/licensecheck#25]
https://9fans.github.io/usr/local/plan9/src/cmd/9pfuse/COPYRIGHT is currently a no match (note: the HTTP server seems to force a download). It seems to be a variant of the MIT license, but it's a variation I can't find elsewhere (e.g. https://fedoraproject.org/wiki/Licensing:MIT https://en.wikipedia.org/wiki/MIT_License https://opensource.org/licenses/alphabetical )

There's a secondary question of whether that "without fee" is a problematic clause or not for FOSS (later variants add commas or say "with or without", to clarify intent). Not sure if that's in scope of this project at all.

Nudge @rsc 

```
The files in this directory are subject to the following license.

The author of this software is Russ Cox.

        Copyright (c) 2006 Russ Cox

Permission to use, copy, modify, and distribute this software for any
purpose without fee is hereby granted, provided that this entire notice
is included in all copies of any software which is or includes a copy
or modification of this software and in all copies of the supporting
documentation for such software.

THIS SOFTWARE IS BEING PROVIDED "AS IS", WITHOUT ANY EXPRESS OR IMPLIED
WARRANTY.  IN PARTICULAR, THE AUTHOR MAKES NO REPRESENTATION OR WARRANTY
OF ANY KIND CONCERNING THE MERCHANTABILITY OF THIS SOFTWARE OR ITS
FITNESS FOR ANY PARTICULAR PURPOSE.

```

**Comment by pombredanne** [comment]
FWIW, this license has been tracked by scancode here as "x11-hanson" https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/x11-hanson.LICENSE

**Comment by rsc** [comment]
This license is the old dtoa.c license (see https://www.netlib.org/fp/dtoa.c), later also used by Plan 9's fmt code. As I remember it, we were able to convince the lawyers that the purpose of releasing the fmt implementation was to provide an alternate dtoa implementation and that it should use the same license. That was no small accomplishment!
Then I used the same license text for some of the code I wrote later on my own, outside of Bell Labs.

It's basically a variant of SPDX HPND and I've added it there.
